release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1539,8 +1559,8 @@ In the following tables, features are marked with the following statuses:
 
 |Cloud controller manager for {alibaba}
 |Technology Preview
-|Technology Preview
-|Technology Preview
+|Removed
+|Removed
```

Comment:
Not sure if we should change `Removed` to `Not Available` to keep consistent with installation.
https://drive.google.com/file/d/1Kqj-rAEFulk9LANhYoFA2fvAqS0eztW3/view?usp=sharing

Reply:
Seems we should, because there is no `Removed` status in the overall part. https://drive.google.com/file/d/1diaLvbDhwX49rvQL-B0QfZ6GVC0qdgPO/view?usp=sharing

Reply:
Good catch - I conflated the two table status options :) Thank you!

Reply:
After talking this through with the RN docs team, we think that the list of accepted statuses for the table should be adjusted so that TP features that become deprecated or removed can have the `Deprecated` or `Removed` status in the tables. The RN lead is going to get PRs up for this soon, so I will leave this as it is in anticipation of that change.